spec/game_over_spec.rb

```diff
@@ -2,18 +2,7 @@
 require './lib/board'
 require './lib/game_over'
 require './spec/player_choice_spec'
-
-class TestUI
-  attr_reader :game_over_win_called, :game_over_draw_called
-
-  def game_over_win(_player)
-    @game_over_win_called = true
-  end
-
-  def game_over_draw
-    @game_over_draw_called = true
-  end
-end
+require './mock/test_ui'
```

Comment:
This isn't directly related to this PR but I think GameOver suffers from multiple responsibilities. You use it to query whether the game is over or not - but it also outputs info. I don't think this is necessary.

Reply:
I see what you're saying. I could make a separate WinLose Class to determine the winner.